modules/security-monitoring-events.adoc

```diff
@@ -5,16 +5,16 @@
 [id="security-monitoring-events_{context}"]
 = Watching cluster events
 
-Cluster administrators are encouraged to familiarize themselves with the _Event_ resource
+Cluster administrators are encouraged to familiarize themselves with the `Event` resource
 type and review the list of system events to
 determine which events are of interest.
 Events are associated with a namespace, either the namespace of the
 resource they are related to or, for cluster events, the `default`
 namespace. The default namespace holds relevant events for monitoring or auditing a cluster,
-such as _Node_ events and resource events related to infrastructure components.
+such as `Node` events and resource events related to infrastructure components.
```

Comment:
Do we want `Node` events or node events?

Reply:
I had done this because the actual thing you grep is "Node", but I can see now that that's confusing. I'll lower it.